samples/unit-tests/maps/map-navigation/demo.js

```diff
@@ -190,3 +190,150 @@ QUnit.test('Map navigation button alignment', assert => {
         'The buttons should be bottom-aligned to the plot box after redraw (#12776)'
     );
 });
+
+QUnit.skip('Orthographic map rotation and panning.', assert => {
+    const getGraticule = () => {
+        const data = [];
+
+        // Meridians
+        for (let x = -180; x <= 180; x += 15) {
+            data.push({
+                geometry: {
+                    type: 'LineString',
+                    coordinates: x % 90 === 0 ? [
+                        [x, -90],
+                        [x, 0],
+                        [x, 90]
+                    ] : [
+                        [x, -80],
+                        [x, 80]
+                    ]
+                }
+            });
+        }
+
+        // Latitudes
+        for (let y = -90; y <= 90; y += 10) {
+            const coordinates = [];
+            for (let x = -180; x <= 180; x += 5) {
+                coordinates.push([x, y]);
+            }
+            data.push({
+                geometry: {
+                    type: 'LineString',
+                    coordinates
+                },
+                lineWidth: y === 0 ? 2 : undefined
+            });
+        }
+
+        return data;
+    };
```

Comment:
Do we need a graticule in a unit test? It's additional tens of thousands of operations that maybe can be skipped?

Reply:
You're right. We need something that makes the initial zoom show the full world, but it doesn't need to be this complex. Either a few lines (equator + a handful of meridians), or some wisely placed map points (poles + some around the equator?) should be enough to make the ortho projection zoom out to the full globe.